src/lib/utils/requestIdleCallback.js

```diff
@@ -0,0 +1,3 @@
+import * as idle from 'requestidlecallback'
```

Comment:
move to `system.web.js` , define the empty fn ( `export const requestIdle = noop`) in `system.native.js`

Reply:
done